nipype/info.py

```diff
@@ -70,27 +70,26 @@ def get_nipype_gitversion():
 # Note: this long_description is actually a copy/paste from the top-level
 # README.txt, so that it shows up nicely on PyPI.  So please remember to edit
 # it only in one place and sync it correctly.
-long_description = \
-    """
+long_description = """\
```

Comment:
Ok, np. It just looked nicer to me this way. Probably not for others, will change XD